src/content/dvt/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Distributed Validator Technology
+description: Distributed Validator Technology enables the distributed operation of an Ethereum validator by multiple parties.
+lang: en
+---
+
+# Distributed Validator Technology {#what-is-dvt}
+_With the advent of Ethreum’s PoS consensus, standard validator setups were accompanied by some challenges and the risk of stake centralization. To help distribute validator nodes and decentralize the network, Distributed Validator Technology was conceived._
+
+<Divider />
+
+## What is DVT? {#what-is-dvt}
+
+**Distributed Validator Technology (DVT)** is a technology primitive in the evolving **Ethereum technology** stack aimed at improving the resilience and health of the Ethereum blockchain. The solution focuses on strengthening the backbone of the blockchain's PoS consensus by enabling multiple non-trusting parties to operate an Ethereum validator collectively, as opposed to how standard validators are operated. This shift in validator operation dramatically improves the liveness and security of the network while significantly reducing slashing, downtime, and centralization risks compared to standard validator setups. 
+
+A way to think about DVT is that DVT is to validators as multisig is to wallets. Where standard validators use a single key to sign duties, the key of a Distributed Validator is split across multiple nodes. In the case of DVT, the "multisig" is operated by a consensus protocol (BFT) instead of human interaction. Enabling the distributed operation of an Ethereum validator with best-in-class security practices. 
+
+Distributing a new or existing validator is quite complex in practice, but using a DVT solution like ssv.network as a layer to split the validator private key into multiple key shares makes the technology more accessible. Regardless of the [DVT-implementation](<https://ssv.network/>) stakers can have multiple nodes each operating a key share in a cluster, allowing them to ensure the uptime of their validators.
```

Comment:
typically we would try not to promote one project when several are available. I think the page would flow better by skipping straight to the next section.
```suggestion
```